msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/base/AbstractDriverBaseTest.java

```diff
@@ -85,7 +84,8 @@ public void openBrowser() {
     }
 
     @AfterMethod
-    public void closeBrowser() {
+    public void teardown() {
+        captureScreen(driver);
         log.info("<<<=== Teardown");
         driver.quit();
```

Comment:
same as browser open. try to minimize open/close the whole browser. you can open new window as example and close the old one. Try to find the best performant way to execute UI tests